docs/source/hardware_and_software_configuration/configuring/configuring_uvc_camera/configuring-uvc-camera.rst

```diff
@@ -4,11 +4,11 @@ Configuring an External UVC Camera and a Powered USB Hub
 Introduction
 ------------
 
-Game rules and in the Game Manual Part 1 have been modified to allow the
-use of USB Video Class (UVC) Compatible Cameras for computer
-vision-related tasks. Teams have the option of using an externally
-connected camera instead of the Android smartphone’s built-in camera for
-computer vision tasks.
+The Competition Manual allows the use of USB Video Class (UVC) compatible
+cameras for computer vision-related tasks. See rule R715 for the full details.
```

Comment:
Rephrase to not point to a specific rule number. Rule numbers may shift year to year.  

Suggest something like:

> Check the Robot Construction rules in the [Competition Manual](https://ftc-resources.firstinspires.org/file/ftc/game/manual) for details.